README.md

```diff
@@ -44,6 +44,8 @@
 - [Consensus Compare: Tendermint BFT vs. EOS dPoS](https://blog.cosmos.network/consensus-compare-tendermint-bft-vs-eos-dpos-46c5bca7204b)
 - [List of projects in Cosmos & Tendermint Ecosystem](https://forum.cosmos.network/t/list-of-projects-in-cosmos-tendermint-ecosystem/243)
 - [The Most Comprehensive Guide to Tendermint Core](https://blockgeeks.com/guides/ultimate-guide-tendermint/)
+- [Randomized Leader Election Proposal](https://research.codechain.io/t/randomized-leader-election-using-vrf/17)
```

Comment:
This is the original proposal, right? Don't think we need it here since the below post contains a link to it anyway.

```suggestion
```